### DISCUSSION TITLE
view youtube videos after embeed

### DISCUSSION THREAD

**Discussion by ViniciusLima7** [requarks/wiki#3891] [Error / Bug Report]
Hello I checked the rendering settings to view embeed video, but even after configuring the video it is not visible.
I realized that using HTML it works
I put a link with details follows
[](https://js.wiki/feedback/p/view-youtube-videos-after-embeed)

**Comment by zblcool** [comment]
Hey! check this out. I made it work by following this answer. and you can customize the CSS by yourself

[https://github.com/requarks/wiki/discussions/4580#discussioncomment-1533379](https://github.com/requarks/wiki/discussions/4580#discussioncomment-1533379)
